go.mod

```diff
@@ -2,4 +2,7 @@ module github.com/livechat/lc-sdk-go/v3
 
 go 1.12
 
-require github.com/livechat/lc-sdk-go/v2 v2.3.0
+require (
+	github.com/google/go-querystring v1.1.0 // indirect
```

Comment:
i am not sure if we should use this package? can't we just pass a map[string]string to Call and then iterate over it to add each key val pair as querystring

Reply:
I wanted to remain simplicity in the code where `POST` requests use structs with `json` tag then `GET` requests use structs with `url` tag. Almost the same standard for `POST` and `GET`, that was my purpose.